.env.development

```diff
@@ -4,6 +4,10 @@ BASE_URL='localhost:1994'
 LMS_BASE_URL='http://localhost:18000'
 LOGIN_URL='http://localhost:18000/login'
 LOGOUT_URL='http://localhost:18000/login'
+LOGO_URL=https://edx-cdn.org/v3/default/logo.svg
```

Comment:
Nit: follow the quoting scheme that the project already uses. It's jarring to see warring styles.

Reply:
counter-nit: this is explicitly copied from the logo-swap doc, which has the https values unquoted.